packages/app-config-writer/test/unit/prompt/smartlinks-config.test.ts

```diff
@@ -54,17 +54,29 @@ describe('Test function getSmartLinksTargetFromPrompt', () => {
             const basePath = join(__dirname, '../../fixtures/no-ui5-deploy-config');
             await expect(getSmartLinksTargetFromPrompt(basePath, loggerMock)).rejects.toThrow();
             expect(debugMock.mock.calls[0][0].message).toContain(`File 'ui5-deploy.yaml' not found in project`);
-            const promptForTarget = promptMock.mock.calls[0][0];
-            expect(promptForTarget).toMatchSnapshot();
-            expect(promptForTarget[0].initial).not.toBeDefined();
+            const urlPrompt = promptMock.mock.calls[0][0][0];
+            expect(urlPrompt.initial).not.toBeDefined();
+            expect(urlPrompt.message).toContain('url');
+            expect(urlPrompt.name).toEqual('url');
+            expect(urlPrompt.type).toEqual('text');
+            expect(urlPrompt.validate).toBeDefined();
+            const clientPrompt = promptMock.mock.calls[0][0][1];
+            expect(clientPrompt.initial).not.toBeDefined();
+            expect(clientPrompt.message).toEqual('SAP client ');
+            expect(clientPrompt.name).toEqual('client');
+            expect(clientPrompt.type).toEqual('text');
+            expect(clientPrompt.format).toBeDefined();
```

Comment:
Thanks.  Follow up suggestion to get it more concise (but just a suggestion, no need to change it)
```
            const [urlPrompt, clientPrompt] = promptMock.mock.calls[0][0];
            expect(urlPrompt.initial).not.toBeDefined();
            expect(urlPrompt.message).toContain('url');
            expect(urlPrompt.validate).toBeDefined();
            expect(urlPrompt).toMatchObject({
                name: 'url',
                type: 'text'
            });
            expect(clientPrompt.initial).not.toBeDefined();
            expect(clientPrompt).toMatchObject({
                message: 'SAP client ',
                name: 'client',
                type: 'text'
            });
            expect(clientPrompt.format).toBeDefined();
```

Reply:
Thanks, since there was a bit of overhaul for the tests, the suggestions were included too